rbd/group.go

```diff
@@ -110,7 +110,8 @@ func GroupImageAdd(groupIoctx *rados.IOContext, groupName string,
 		cephIoctx(groupIoctx),
 		cGroupName,
 		cephIoctx(imageIoctx),
-		cImageName)
+		cImageName,
+		C.uint32_t(0))
```

Comment:
https://github.com/ceph/ceph/blob/reef/src/include/rbd/librbd.h#L1436

I don't understand these changes to `rbd_group_image_add`. It only has the four arguments that we're already passing.